<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="MAVBCP532" ignored="0" name="Report Scenarios" tests="2" failures="1" timestamp="2020-06-28T19:04:16 IST" time="200.329" errors="0">
  <testcase name="method1" time="121.262" classname="com.testNGassignment.TestNG_AssertionReports"/>
  <testcase name="method2" time="73.413" classname="com.testNGassignment.TestNG_AssertionReports">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.testNGassignment.TestNG_AssertionReports.method2(TestNG_AssertionReports.java:24)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- method2 -->
</testsuite> <!-- Report Scenarios -->
